packages/core/admin/ee/server/bootstrap.js

```diff
@@ -14,9 +15,15 @@ module.exports = async () => {
   }
 
   if (features.isEnabled('audit-logs')) {
+    await reserveTablesWithPrefix('strapi_audit_logs');
+
     await actionProvider.registerMany(actions.auditLogs);
   }
 
+  if (features.isEnabled('review-workflows')) {
+    await reserveTablesWithPrefix('strapi_workflows');
+  }
+
```

Comment:
Might not need that in main for now. 🤔 
What do you think of adding those lines in review workflow PR instead when this current PR will be merged in main?

Reply:
Good point. Okay I will revert this change and make a TODO in the RW feature PR